PROTOCOL.md

```diff
@@ -2023,6 +2310,91 @@ Example (for illustration purposes; each JSON object must be a single line in th
 }
 ```
 
+### CDF Files
+
+#### Add File
```

Comment:
What's the difference between an Add file vs CDF file? By reading this spec it does not give me that information and the two looks identical.

Reply:
Good question. For the fields, there are no differences, including `RemoveFile`.

The difference is for the connector to tell the [_change_type](https://docs.databricks.com/delta/delta-change-data-feed.html#change-data-event-schema) in the response. It would be `insert` for AddFile and `delete` for RemoveFile, and the value is included in the CDFFile for each row.

Reply:
Just for my education: connector will decide change_type based on "add" and "remove". How would the "cdf" file be interpreted? I did not quite follow what `and the value is included in the CDFFile for each row.` mean.